packages/pn-commons/src/components/SnackBar/SnackBar.tsx

```diff
@@ -86,7 +89,7 @@ const SnackBar = ({
               }}
               variant={variant}
             >
-              {title && <AlertTitle>{title}</AlertTitle>}
+              {title && <AlertTitle id={`alert-${id}`}>{title}</AlertTitle>}
```

Comment:
@leleOFA perchè l'id va in questa condizione? non va bene alla riga 77? se il titolo manca noi perdiamo l'id di questo item html

Reply:
i ragazzi hanno richiesto l'id sul titolo, non sulla snackbar